CosmosToRedis/CosmosToRediscs.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using StackExchange.Redis;
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+using Container = Microsoft.Azure.Cosmos.Container;
+
+namespace CosmosToRedis
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+
+    internal class CosmosToRediscs
+    {
+        //redis connection string
+        static ConnectionMultiplexer redisconnect = ConnectionMultiplexer.Connect("PrimaryConnectionString");
+        static IDatabase cache = redisconnect.GetDatabase();
+        public const string localhostSetting = "redisLocalhost";
+
+        [FunctionName("CosmosToRedis")]
+        public static void Run([CosmosDBTrigger(
+        databaseName: "dbname",
+        containerName: "containername",
+        Connection = "endpoint",
+        LeaseContainerName = "leases")]IReadOnlyList<ListData> input, ILogger log)
+        {
+            if (input == null || input.Count <= 0) return;
+
+            var cache = redisconnect.GetDatabase();
```

Comment:
It seems like you already define cache one line 33. Any reason for doing it again? If so, maybe we don't have to initiate it in line 33 yet.

Reply:
nit: change `var` to `IDatabase`